roles/vendor/redis/templates/redis_cluster.service

```diff
@@ -2,11 +2,12 @@
 Description=Redis key-value database on {{ redis_port }}
 After=network.target
 [Service]
-ExecStart=/usr/local/bin/redis-server redis_{{ redis_port }}.conf --supervised systemd --daemonize no
+ExecStart=/usr/bin/redis-server redis_{{ redis_port }}.conf --supervised systemd --daemonize no
+ExecStop=/usr/bin/redis-cli -h 127.0.0.1 -p 7000 shutdown
 Type=notify
 User=redis
 Group=redis
-WorkingDirectory=/etc/redis_cluster/redis_{{ redis_port }}
+WorkingDirectory=/etc/{{redis_dir}}/redis_{{ redis_port }}
 LimitNOFILE=65535
 Restart=always
 
```

Comment:
This seems to be identical to `redis.service` with only some variables different. Please merge them (you can just call the same `template` module with different options.